app/screens/manage_channel_members/manage_channel_members.tsx

```diff
@@ -231,25 +235,44 @@ export default function ManageChannelMembers({
 
     useNavButtonPressed(MANAGE_BUTTON, componentId, toggleManageEnabled, [toggleManageEnabled]);
 
-    useEffect(() => {
-        mounted.current = true;
-        const options: GetUsersOptions = {sort: 'admin', active: true, per_page: PER_PAGE_DEFAULT};
+    const handleReachedBottom = useCallback(() => {
+        if (hasMoreProfiles.current && !loading && !searchedTerm) {
+            pageRef.current += 1;
+            setLoading(true);
+            getFetchChannelMembers();
+        }
+    }, [loading, searchedTerm]);
```

Comment:
This also depends on `getFetchChannelMembers`. We don't expect that to change during the lifetime of the component, but let's add it to be "future proof".